projects/notebooks/testing/config.yaml

```diff
@@ -257,12 +269,12 @@ tests:
     # The test flavor to run. Can be: ods-ci, notebook-performance or gating
     test_flavor: ods-ci
     identifier: unspecified
-    minio_bucket_name: notebooks
+    minio_bucket_name: ds-pipeline-dspa
     # number of test repetition to perform
     repeat: 1
     users:
       # number of users to simulate
-      count: 4
+      count: 5
```

Comment:
this, and below, and above (dedicated) shouldn't be changed, so that we can merge the PR